Cargo.toml

```diff
@@ -26,4 +26,11 @@ description = "A library for safe, in-place construction of Rust (and C++!) obje
 
 [features]
 alloc = []
+cxx_support = [ "cxx" ]
 default = ["alloc"]
+
+[dependencies]
+cxx = { version = "1", optional = true }
```

Comment:
I am not a fan of dtlonay's odd theory of versions for this kind of thing... specify a minor version too? Also, for optional deps, I tend to prefer

```toml
[dependencies.cxx]
version = "1.0"
optional = true
```

Reply:
Done